Feature/175873968 add suggester for ontology labels (#28)
ebi-gene-expression-group/index-scxa: README.md

You can override the default target Solr collection name by setting `SOLR_COLLECTION`, but remember to include the additional `v<schema-version-number>` at the end, or the loader might refuse to load this.

> ## Add suggesters

Reviewer: Is it needed to add the suggesters right after creating the schema or this should go after experiments data has been loaded? Or it doesn't matter? If it should happen after experiments are created, then I suggest that the docs for this moves after the part where we load experiments.

> For Single Cell Expression Atlas run:

Reviewer: ```suggestion
For Single Cell Expression Atlas run:

```

```bash
create-scxa-analytics-suggesters.sh
```

> ## Suggesters Dictionary Implementation
>  We are using multiple dictionaries(dictionaryImpl) for a single SuggestComponent to fetch good number of suggestions.
>  #### Here is the Dictionary Implementations: 
>    - ontologyAnnotationSuggester

Reviewer: ```suggestion

## Suggesters Dictionary Implementation

We are using multiple dictionaries (dictionaryImpl) for a single `SuggestComponent` to fetch various suggestions.

#### Dictionary Implementations: 

   - ontologyAnnotationSuggester
```

To improve readability.
